hawc/apps/lit/managers.py

```diff
@@ -559,6 +559,11 @@ def global_df(self) -> pd.DataFrame:
         ).values_list(*mapping.values())
         return pd.DataFrame(list(qs), columns=list(mapping.keys()))
 
+    def full_text_search(self, search_text):
+        return self.annotate(search=constants.REFERENCE_SEARCH_VECTOR).filter(
+            Q(search=search_text) | Q(identifiers__unique_id=search_text)
```

Comment:
I'd pull the `| Q(identifiers__unique_id=search_text)`, and maybe add a **filters argument to the func, and then add this additional item via a func parameter

Reply:
I was thinking something like this:

```python
def full_text_search(self, search_text, filters=None):
    if filters is None:
         filters = !()
    filters |= Q(search ...)
         
    return self.annotate(search=constants.REFERENCE_SEARCH_VECTOR).filter(filters)
```

Since i would expect a full_text_search to also look at IDs necessarily